### ISSUE TITLE
Separate Autoplay for a single video

### ISSUE THREAD

**Issue by limonovthesecond2** [libre-tube/LibreTube#2096]
### Describe your suggested feature

Create a new preference to auto-playing when there is only **one** video in a queue. Therefore, rename the existing "Autoplay", for example, to "Autoplay for queue". Using this feature with disabled "Insert related videos" would be very helpful for reasons below. 

### What is it for?
1. A user doesn't always want to watch a video immediately after opening it. He may want to just read the description, comments, add the video to a queue and only then start playback.
2. Using mobile network, it would be awesome to save bits without automatic playing videos. After reading the description, the user may not want to play the video or play it in "Audio only" mode to save bits.
3. This feature is required to use this software on poor networks (2G for example), but then also required integration with "Data saver".

### Other details

It would be nice to also implement a "Data saver" to not load any thumbnail in player screen (after loading a video or scrolling it) before clicking play **video** button. Users using "Data saver" don't need thumbnails at all, and therefore they usually use audio-only mode. Without switchable video Autoplay and fully disabled thumbnails, it's impossible to listen to videos with LibreTube on very poor networks.

### How can be implemented with "Data saver"

Insert related videos is off, Data saver is off, Autoplay is on: The video loads with thumbnail and does Autoplay.
Insert related videos is off, Data saver is off, Autoplay is off: The video loads with thumbnail and doesn't Autoplay.
Insert related videos is off, Data saver is on, Autoplay is on: The video loads with thumbnail due to Autoplay and does Autoplay.
Insert related videos is off, Data saver is on, Autoplay is off: The video loads without thumbnail and doesn't Autoplay.

Insert related videos is on, Data saver is off, Autoplay for queue is on: Videos load with thumbnails and do autoplay.
Insert related videos is on, Data saver is off, Autoplay for queue is off: Videos load with thumbnails and don't autoplay.
Insert related videos is on, Data saver is on, Autoplay for queue is on: Videos load with thumbnail due to autoplay and do autoplay.
Insert related videos is on, Data saver is on, Autoplay for queue is off: Videos load without thumbnail and don't autoplay.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by GET100PERCENT** [comment]
I think it will be easier if there is another button "See Video Description" here, after long pressing a video.
![Screenshot_20221127-212141](https://user-images.githubusercontent.com/116144024/204144487-ec458030-17d3-4069-8d63-20a86f04cc41.jpg)

**Comment by limonovthesecond2** [comment]
> #2198 might be more simple implementation. Yours feels like too complex.

I agree. Adding logic to create a queue only after playback has started would be much better.
